src/setup-beam.js

```diff
@@ -303,7 +353,7 @@ function getVersionFromSpec(spec, versions) {
     }
   }
 
-  return version
+  return version === null || version === undefined ? null : version
```

Comment:
Bug fix: this would otherwise mean we'd try to install version `undefined`. Since we're already matching against `null` this is the choice I preferred.